pylua/timer.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# timer works with milliseconds
```

Comment:
@dtrunov, this is wrong place for documentation. Use doxygen.
See python sample here http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html